docs/reference/proto-definition.md

```diff
@@ -17,8 +17,10 @@ A field declaration has the following syntax:
 field fieldType fieldName defaultValue
 ```
 
-`field` is a reserved keyword, `fieldType` is one of: `SFNode, SFColor, SFFloat, SFInt32, SFString, SFVec2f, SFVec3f, SFRotation, SFBool, MFNode, MFColor, MFFloat, MFInt32, MFString, MFVec2f` and `MFVec3f`.
-`fieldName` is a freely chosen name for this field and `defaultValue` is a literal default value that depends on `fieldType`.
+* `field` is a reserved keyword,
+* `fieldType` is one of: `SFNode, SFColor, SFFloat, SFInt32, SFString, SFVec2f, SFVec3f, SFRotation, SFBool, MFNode, MFColor, MFFloat, MFInt32, MFString, MFVec2f` and `MFVec3f`.
```

Comment:
Could you please also apply this change?
This list is not up-to-date and `MFBool` and `MFRotation` are missing.
```suggestion
* `fieldType` is one of: `SFNode, SFColor, SFFloat, SFInt32, SFString, SFVec2f, SFVec3f, SFRotation, SFBool, MFNode, MFColor, MFFloat, MFInt32, MFString, MFVec2f`, `MFVec3f`, `MFRotation` and `MFBool`.
```